opentelemetry-proto/Cargo.toml

```diff
@@ -48,7 +48,7 @@ with-schemars = ["schemars"]
 with-serde = ["serde", "hex"]
 
 [dependencies]
-tonic = { workspace = true, optional = true, features = ["codegen", "prost"] }
```

Comment:
Since we are using workspace dependencies can we disable it in workspace?

Reply:
@sandhose Could you make this suggestion?